app/policies/podcast_policy.rb

```diff
@@ -1,14 +1,14 @@
 class PodcastPolicy < ApplicationPolicy
   def create?
-    token && token.authorized?(account_id)
+    token&.authorized?(account_id, :podcast_create)
```

Comment:
Not exactly tied to this line of code ... but just to think through the way we'll use auth-Feeder on day 1 ...

1. CMS distributions doing synchronous creates of podcasts/episodes.  This will get the account tied to [the series](https://github.com/PRX/cms.prx.org/blob/master/app/models/distributions/podcast_distribution.rb#L51) or [the story's series](https://github.com/PRX/cms.prx.org/blob/master/app/models/story_distributions/episode_distribution.rb#L56).  And since those aren't requesting any scope yet, they should just continue to get all scopes, and work.
2. `publish.prx.org` users updating podcasts/episodes.  But the JWTs already contain all the feeder specific scopes, so things should just continue to work as-is.
3. Announcement/SNS based updates.  Which don't rely on account auth at all.

Reply:
Oh, and there's a (4) ...

Castle requesting the wildcard token to scrape all Feeder podcasts/episodes.  /me looks at that code.

Reply:
Looks like it's a [wildcard read-private token](https://github.com/PRX/castle.prx.org/blob/master/lib/feeder/api.ex#L12) ... and since that won't be namespaced (i think?) it should just work against these changes.  Right?

Reply:
Yes, it should. Unnamespaced scopes are treated as `*:scope` which means the token would be equivalent to `*:'*:read-private'` which should work!